quickstarts/router-ext/pom.xml

```diff
@@ -15,7 +15,7 @@
         <dependency>
             <groupId>org.kie.server</groupId>
             <artifactId>kie-server-router-proxy</artifactId>
-            <version>7.51.0-SNAPSHOT</version>
+            <version>7.59.0.Final-redhat-00006</version>
```

Comment:
the latest upstream version is 7.65.0.Final. If we need to use SNAPSHOTs that would be 7.66.0-SNAPSHOT.

Reply:
I've keeping using the latest released version (product)